typescript/sdk/src/ccip/utils.ts

```diff
@@ -121,7 +121,6 @@ export async function isSupportedCCIPLane({
 }): Promise<boolean> {
   const originRouter = getCCIPRouterAddress(origin);
   const destinationSelector = getCCIPChainSelector(destination);
-
```

Comment:
Highly recommend using `git add -p` to conciously check in the desired changes

Reply:
this was intended to remove extraneous whitespace in a small function, but can revert if we want to minimise diff